lab-tjay/route/minion-router.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const Minion = require('../model/minion.js');
+
+let minionRouter = module.exports = new Router();
+
+minionRouter.post('/api/minions', jsonParser, (req, res, next) => {
```

Comment:
The paths for all the methods on a single REST resource should be consistent. I know that is a bit annoying grammatically, what with the whole plurals thing, but naming them the same will save you much troubles. It's also the standard convention.